concepts/05 Widgets/Scheduler/050 Timetable.md

```diff
@@ -63,6 +63,89 @@ The **Scheduler** widget allows you to customize its timetable. You can specify
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+            :current-date="currentDate"
+            :start-day-hour="8"
+            :end-day-hour="19"
+            :cell-duration="60"
+            :first-day-of-week="1"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxScheduler } from 'devextreme-vue/scheduler';
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: [{
+                        text: 'Website Re-Design Plan',
```

Comment:
Each next nesting level should be indented by 4 spaces, not by 8 as on this line.